manifests/role/tools_lib/test.pp

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2019 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# stand up apache server w/ dependancies for confluence and apache
+#
+# vhost name is: test.tools.lib.umich.edu
+#
+#
+# @example
+#   include nebula::role::tools_lib::test
+class nebula::role::tools_lib::test {
+
+  class { 'nebula::profile::tools_lib2':
```

Comment:
the listed profile doesn't exist; remove this role?

Reply:
deleted